src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -719,27 +719,23 @@ If you need to send a response to the invoker of the function, such as a user, c
 The response information **will not be logged** to the Appwrite Console. 
 There are several possible ways to send a response, explore them in the following Appwrite Function.
 
-### Response Types {% #response-types %}
+### Response types {% #response-types %}
 
 {% table %}
 * Response
 * Description
 ---
-* `text`
-* Converts the body using UTF-8 encoding into a binary Buffer.
----
-* `binary`
-* Packages binary bytes, the status code, and the headers into an object.
```

Comment:
Sorry, let's keep it in the table, but not add it to the code yet. Let's put it after json.